www/js/services.js

```diff
@@ -1,96 +1,243 @@
-angular.module('starter.services', [])
+angular.module('mobility.services', [])
 
-.factory('Chats', function() {
-  // Might use a resource here that returns a JSON array
-
-  // Some fake testing data
-  var chats = [{
+.factory('Incidents', function() {
+  // Traffic API mock data
+  var incidents = [{
```

Comment:
Finally, someone with mock data! :tada: :tada: 

Two things:

* Going forward, it'd be cool if you moved this data into a folder outside of www where you can run `http-server`. Then, you can emulate actually communicating with the server. But ...
* It'd be better if you could communicate this with @eunjoolee787 and @fobabett so they can create a mock endpoint for you with this data. Then, you can download their project and run it locally as your source of data.

This has no bearing on whether or not your PR gets approved. Just wanted to point those two things out as next steps.